modules/nf-core/mapad/map/meta.yml

```
@@ -0,0 +1,81 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "mapad_map"
+description: Map short-reads to an indexed reference genome
+keywords:
+  - mapad
+  - short-read
+  - align
+  - reference
+  - fasta
+  - bam
+  - map
+  - fastq
+tools:
+  - "mapad":
+      description: "An aDNA aware short-read mapper"
+      homepage: "https://github.com/mpieva/mapAD"
+      documentation: "https://github.com/mpieva/mapAD"
+      tool_dev_url: "https://github.com/mpieva/mapAD"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: BAM/CRAM/FASTQ/FASTQ.GZ file
+      pattern: "*.{bam,cram,fastq,fastq.gz}"
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing reference information
+        e.g. [ id:'test', single_end:false ]
+  - index:
+      type: file
+      description: mapAD genome index files
+      pattern: Directory containing mapAD index files *.{tbw,tle,toc,tos,tpi,trt,tsa}
+  - mismatch_parameter:
+      type: float
+      description: |
+        Minimum probability of the number of mismatches under `-D` base error rate
+  - double_stranded_library:
+      type: boolean
+      description: Library preparation method
+      pattern: "true or false"
+  - five_prime_overhang:
+      type: float
+      description: 5'-overhang length parameter
+  - three_prime_overhang:
+      type: float
+      description: 3'-overhang length parameter
+  - deam_rate_double_stranded:
+      type: float
+      description: Deamination rate in double-stranded stem of a read
+  - deam_rate_single_stranded:
+      type: float
+      description: Deamination rate in single-stranded ends of a read
+  - indel_rate:
+      type: float
+      description: Expected rate of indels between reads and reference
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - bam:
+      type: file
+      description: BAM file
```

```suggestion
      description: BAM file
```

or reads against the reference genome or something (https://nf-co.re/docs/contributing/modules#documentation point 10)